api/service.go

```diff
@@ -652,47 +825,73 @@ func (s *service) GetFile(ctx context.Context, req *pb.GetFileRequest) (*pb.GetF
 	if !ok {
 		log.Fatal("scope required")
 	}
-	file, err := s.getFileWithScope(ctx, req.Path, scope)
+	folderName, fileName := parsePath(req.Path)
+	folder, err := s.getFolderWithScope(ctx, folderName, scope)
+	if err != nil {
+		return nil, err
+	}
+	pth := path.New(filepath.Join(folder.Path, fileName))
+	if err = pth.IsValid(); err != nil {
+		return nil, err
+	}
+
+	node, err := s.ipfsClient.Unixfs().Get(ctx, pth)
```

Comment:
So this traverses the `Links` added with the `Object().AddLink` and resolves the path starting from the project folder?

Reply:
This will just get the node at the path. If it's a directory, you can call `Ls` on it to list it links.